pkg/containers/utils.go

```diff
@@ -0,0 +1,83 @@
+package containers
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"syscall"
+)
+
+// runsOnContainerV1 tests if process is running on a container in cgroupsv1
+// it tests by checking for existance of a release_agent file in cpuset
+// from https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt:
+//
+// - release_agent: the path to use for release notifications (this file exists in the top cgroup only)
+//
+// WARNING: IF THIS WASN'T CLEAR THIS IS ONLY VALID FOR CGROUPSV1 MACHINES
+func runsOnContainerV1() (bool, error) {
+	const releseAgentPath = "/sys/fs/cgroup/cpuset/release_agent"
+	_, err := os.Stat(releseAgentPath)
+	if os.IsNotExist(err) {
+		return true, nil
+	}
+	if err != nil {
+		return false, err
+	}
+	return false, nil
+}
+
+// mountsCpuset mounts /sys/fs/cgroup/cpuset to the container's cpuset fs
+// this is done in order to detect host cgroup data when running tracee from a container in cgroupsv1
+func mountCpuset(mountPath string) error {
+	isMp, _ := isMountpoint(mountPath)
+
+	if isMp {
+		return nil
+	}
+
+	// equivalent to mount -t cgroup -o cpuset cgroup /sys/fs/cgroup/cpuset
```

Comment:
```suggestion
	// equivalent to mount -t cgroup -o cpuset cgroup cpuset
```

Reply:
actually should be `mountPath` in the end.